RLBotCS/ManagerTools/Logging.cs

```diff
@@ -0,0 +1,118 @@
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace RLBotCS.ManagerTools;
+
+public class Logging : ILogger
+{
+    private const string Grey = "\x1b[38;20m";
+    private const string LightBlue = "\x1b[94;20m";
+    private const string Yellow = "\x1b[33;20m";
+    private const string Green = "\x1b[32;20m";
+    private const string Red = "\x1b[31;20m";
+    private const string BoldRen = "\x1b[31;1m";
+    private const string Reset = "\x1b[0m";
+
+    private static readonly LogLevel LoggingLevel = LogLevel.Information;
+    private static readonly string AppName = "RLBotServer".PadLeft(12);
+
+    private readonly string _name;
+    private readonly LogLevel _minLevel;
+    private static readonly object _lock = new object();
+
+    public Logging(string name, LogLevel minLevel)
+    {
+        _name = name.PadLeft(20);
+        _minLevel = minLevel;
+    }
+
+    public IDisposable? BeginScope<TState>(TState state)
+        where TState : notnull => null;
+
+    public bool IsEnabled(LogLevel logLevel) => logLevel >= _minLevel;
+
+    public void Log<TState>(
+        LogLevel logLevel,
+        EventId eventId,
+        TState state,
+        Exception? exception,
+        Func<TState, Exception?, string> formatter
+    )
+    {
+        if (!IsEnabled(logLevel))
+            return;
+
+        var logLevelColors = GetLogLevelColors(logLevel);
+        var logLevelString = GetLogLevelString(logLevel);
+        var message = formatter(state, exception);
+
+        lock (_lock)
+        {
+            var logBuilder = new StringBuilder();
+            logBuilder.Append(
+                $"{logLevelColors[0]}{DateTime.Now:yyyy-MM-dd HH:mm:ss}{Reset} "
```

Comment:
Including the date is a bit excessive. Let's save some space
```suggestion
                $"{logLevelColors[0]}{DateTime.Now:HH:mm:ss}{Reset} "
```